packages/hooks/use-draggable/src/index.ts

```diff
@@ -0,0 +1,95 @@
+import type {MoveMoveEvent, MoveResult} from "@react-aria/interactions";
+
+import {useEffect, useRef} from "react";
+import {useMove} from "@react-aria/interactions";
+
+export interface UseDraggableProps {
+  /**
+   * Ref to the moving target DOM node.
+   */
+  targetRef?: React.RefObject<HTMLElement>;
+  /**
+   * Ref to the drag DOM node.
+   */
+  dragRef?: React.RefObject<HTMLElement>;
+  /**
+   * Whether the target is draggable.
+   * @default true
+   */
+  draggable?: boolean;
+  /**
+   * Whether the target can overflow the viewport.
+   * @default false
+   */
+  overflow?: boolean;
```

Comment:
To maintain consistency with other hooks in the project, I suggest renaming `draggable` to `isDisabled` and `overflow` to `isOverflow` or `canOverflow`.

Reply:
ACK!
I have renamed `draggable` to `isDisabled` and `overflow` to `canOverflow`.